tests/stress_tests/scripts/compare_memcheck_2_runs.py

```diff
@@ -14,16 +14,28 @@
 import csv
 import json
 import os
+import sys
 from collections import OrderedDict
 from glob import glob
 from operator import itemgetter
 from pathlib import Path
+import logging as log
+
+import pandas
+from scipy.stats import gmean
```

Comment:
Could limit dependencies for run_memcheck.py compare. I'm OK to have dependencies for advanced analytics/reporting as long as those are optional.

Reply:
I moved it under a function, so import will be done only after call of a function (so it will work as optional)